lib/transbank/sdk/patpass/errors/integration_type_error.rb

```diff
@@ -0,0 +1,15 @@
+module Transbank
+  module Patpass
+    module PatpassByWebpay
+      module Errors
+        class IntegrationTypeError < ::Transbank::Patpass::Errors::PatpassError
```

Comment:
If the namespace is `Transbank::Patpass::PatpassByWebpay::Errors` then this file is in the incorrect directory

Reply:
Fixed!